cmd/krel/cmd/root.go

```diff
@@ -57,19 +55,3 @@ func init() {
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
 }
-
-func cleanupTmpDir(dir string) error {
```

Comment:
`deadcode`?

Reply:
Yes, I could also add a `nolint` if we will need it in the future. Right now the git package takes care of that cleanup.

Reply:
We'll need a common cleanup function somewhere, that is separate from the git package.
So this is fine right now, but we'll need to address that in a follow-up.